services/server/src/sourcify-chains-default.json

```diff
@@ -127,7 +127,30 @@
   },
   "560048": {
     "sourcifyName": "Ethereum Hoodi Testnet",
-    "supported": true
+    "supported": true,
+    "etherscanApi": {
+      "apiURL": "https://api-hoodi.etherscan.io",
+      "apiKeyEnvName": "ETHERSCAN_API_KEY"
+    },
+    "fetchContractCreationTxUsing": {
+      "blockscoutApi": {
+        "url": "https://eth-hoodi.blockscout.com/"
+      },
+      "etherscanApi": true,
+      "routescanApi": {
+        "type": "testnet"
+      }
+    },
+    "rpc": [
+      "https://rpc.hoodi.ethpandaops.io",
```

Comment:
Doesn't this need authorization headers?

Reply:
Nope, Hoodi and Holesky RPCs are open. Only mainnet is with Auth

Reply:
Alright, just wanted to make sure :)